.cirrus.yml

```diff
@@ -127,29 +127,24 @@ task:
 
 task:
   matrix:
-    # include minor version in vanilla images
-    - name: openbsd-7-1-vanilla
-      env:
-        PACKERFILE: packer/bsd/openbsd_vanilla.pkr.hcl
-
-    # but not in images used for CI
-    - name: openbsd-7-postgres
-      depends_on:
-        - openbsd-7-1-vanilla
-      env:
-        PACKERFILE: packer/bsd/openbsd_postgres.pkr.hcl
+    - env:
+        BUILD_TYPE: vanilla
+      matrix:
+        - name: openbsd-vanilla
+        - name: netbsd-vanilla
 
-    # see openbsd case
-    - name: netbsd-9-2-vanilla
-      env:
-        PACKERFILE: packer/bsd/netbsd_vanilla.pkr.hcl
+    - env:
+        BUILD_TYPE: postgres
+      matrix:
+        - name: openbsd-postgres
+          depends_on:
+            - openbsd-vanilla
+        - name: netbsd-postgres
+          depends_on:
+            - netbsd-vanilla
 
-    # see openbsd case
-    - name: netbsd-9-postgres
-      depends_on:
-        - netbsd-9-2-vanilla
-      env:
-        PACKERFILE: packer/bsd/netbsd_postgres.pkr.hcl
+  env:
+    PACKERFILE: packer/bsd/bsd.pkr.hcl
 
```

Comment:
The only difference to the other packer invocation now is that -var bucket  is defined for net/openbsd, but not for the others. Could we, possibly in a separate PR, combine the packer invocations?